index.d.ts

```diff
@@ -7,6 +7,8 @@ declare module 'react-use-cookie' {
   interface cookieOptions {
     days?: number;
     path?: string;
+    samesite?: 'none' | 'lax' | 'strict';
```

Comment:
Users should be recommended to use useful values:
`'lax' | 'strict' | 'none'`
Also add a link to https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite